pom.xml

```diff
@@ -19,11 +19,25 @@
 	</properties>
 
 	<dependencies>
-		<dependency>
+	<!--	<dependency>
 			<groupId>org.springframework.boot</groupId>
```

Comment:
Git keeps history so in general, there is no need to comment something out. You are safe to delete it and then if you need that piece of code you can look in git history.